src/Files.App/Dialogs/CompressArchiveDialog.xaml

```diff
@@ -0,0 +1,169 @@
+﻿<ContentDialog
+	x:Class="Files.App.Dialogs.CompressArchiveDialog"
```

Comment:
Should this be called "CreateArchiveDialog"?

Reply:
It's done.